experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/tsconfig.json

```diff
@@ -1,13 +1,9 @@
 {
-	"extends": ["../../../../../../common/build/build-common/tsconfig.test.json"],
+	"extends": "../../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
 		"rootDir": "./",
 		"outDir": "../../lib/test",
 		"types": ["node", "jest"],
```

Comment:
node and jest don't really belong together, can node go away?

Reply:
The test are using 'assert' and 'fail' from node.